src/components/Home.tsx

```diff
@@ -32,42 +32,65 @@ const Flex = styled.div<FlexboxProps>`
   ${flexbox};
 `;
 
-const PageLink = styled(NavLink)<TypographyProps>`
+const ManifestoLink = styled(NavLink)<TypographyProps & FlexboxProps>`
   ${typography};
+
+  display: flex;
+  ${flexbox}
+`;
+
+const DoubleLineTextImage = styled.img`
+  height: 10vw;
+  max-height: 88px;
+`;
+const SingleLineTextImage = styled.img<SpaceProps>`
+  height: 5vw;
+  max-height: 44px;
+
+  ${space};
 `;
 
-const Img = styled.img<LayoutProps & SpaceProps>`
-  max-width: 30%;
+const ProjectsLink = styled(NavLink)<TypographyProps & FlexboxProps>`
+  ${typography};
+
+  display: flex;
+  align-self: center;
+  align-items: center;
```

Comment:
Approved! with one small change here.

As with the stack overflow solve, let's update `align-items` to `flex-start`. I've just tested it on Safari and it's looking fresssh

Reply:
Weird on my safari `center` is working. `flex-start` working too, will change to that! It's definitely a quirky css bug!